### ISSUE TITLE
No response after clicking on “Download”

### ISSUE THREAD

**Issue by GrimPixel** [louisgv/local.ai#98]
After selecting a model and clicking on “Download”, nothing happened but having a tick by the model description, and the fan starts running fast.
It's the Appimage version.

**Comment by navr32** [comment]
I have the same too !  The model is download in backend..so if you monitor your file directory you will see the file come in and growing..when the file is full here...no network download and file stop to growing i stop and restart localai and the model is loaded...and you have just to check it with the button on the right...if the model is not full download you click on the left pause button and after on the play button and the download resume and finish if you have stop localai before it finish download...

**Comment by GrimPixel** [comment]
Ah, today the entry of the model appears with about 80% download progress.
So it was being downloaded without the entry showing up.

**Comment by navr32** [comment]
Yes the app need improvement on the UI to effectively show the download state..and so and so and so ..but a very very early app..just come to born...so be patient ..and you can add pull request to ask UI showview or state view of the download state..

**Comment by GrimPixel** [comment]
It is a good habit to tell people the problems I discovered...before spending much time understanding this code ...I am patient as usual.
I clicked on a new model to download it, then clicked that “Refresh Models Directory” button at the top left corner right afterwards. Nothing happened.
